test/functional/ui/screen.lua

```diff
@@ -463,9 +479,9 @@ function Screen:_handle_mode_change(mode, idx)
 end
 
 function Screen:_handle_set_scroll_region(top, bot, left, right)
-  self._scroll_region.top = top
+  self._scroll_region.top = top + 1
```

Comment:
@bfredl This is causing many tests to fail... Why are we changing it? We would also need to make the change in `api/ui.c`, right?